### PR TITLE
feat: Implement unsupported SSG version warning

### PR DESCRIPTION
**Pull request by bastilian**
This introduces a `UnsupportedSSGVersion` component to show as a warning in various places:

Screenshot 2020-11-17 at 04 33 39

Screenshot 2020-11-17 at 04 29 35

Screenshot 2020-11-17 at 04 29 55

### CODE REVIEW

File: src/PresentationalComponents/ReportsTable/Cells.js [hunk 1]
```diff
@@ -21,33 +21,56 @@ export const Name = (profile) => (
...
+        8: 'purple',
+        default: 'var(--pf-global--disabled-color--200)'
+    };
+    const color = rhelColorMap[majorOsVersion] || rhelColorMap.default;
+
+    return <React.Fragment>
+        <Label { ...{ color } }>RHEL { majorOsVersion }</Label>
+        { ssgVersion && <Text>
+            <GreySmallText>
+                <SSGVersionWarning supported={ supported }>{ ssgVersion }</SSGVersionWarning>
```
Comment (bastilian) [inline src/PresentationalComponents/ReportsTable/Cells.js:39]: For this warning to be activated the `supported` field needs to be requested from in GraphQL[1]

[1] https://github.com/RedHatInsights/compliance-frontend/blob/master/src/SmartComponents/SystemsTable/SystemsTable.js#L55

File: src/PresentationalComponents/ReportsTable/Cells.js [hunk 2]
```diff
@@ -21,33 +21,56 @@ export const Name = (profile) => (
...
         <Progress
             measureLocation={ 'outside' }
             value={ (100 / totalHostCount) * compliantHostCount } />
-        <GreySmallText>{ `${ compliantHostCount } of ${ totalHostCount } systems` }</GreySmallText>
-    </React.Fragment>
-);
+        <GreySmallText>
+            { `${ compliantHostCount } of ${ totalHostCount } systems ` }
+
+            { unsupportedSystems > 0 && <SSGVersionWarning
```
Comment (bastilian) [inline src/PresentationalComponents/ReportsTable/Cells.js:62]: For this warning a `unsupportedSystems` field is required in the backend and needs to be requested[1].

[1] https://github.com/RedHatInsights/compliance-frontend/blob/master/src/SmartComponents/Reports/Reports.js#L17

File: src/SmartComponents/ReportDetails/ReportDetails.js [hunk 3]
```diff
@@ -95,11 +96,14 @@ export const ReportDetails = () => {
...
     }, ...showSsgVersions ? [{
-        key: 'facts.compliance.ssg_version',
+        key: 'facts.compliance.profiles',
         title: 'SSG version',
         props: {
             width: 5
-        }
+        },
+        renderFunc: (profiles) => (
+            <Cells.SSGVersion { ...{ profiles } } />
```
Comment (bastilian) [inline src/SmartComponents/ReportDetails/ReportDetails.js:105]: For the warning to appear in the report details for the systems the query change in the systems table should cover this.

File: src/SmartComponents/ReportDetails/ReportDetails.js [hunk 4]
```diff
@@ -164,6 +168,13 @@ export const ReportDetails = () => {
                                 />
                             </div>
                         </div>
+                        {  profile.unsupportedSystems > 0 && <Text>
```
Comment (bastilian) [inline src/SmartComponents/ReportDetails/ReportDetails.js:171]: For the warning in the Report details header the `unsupportedSystems` field needs to be requested from GraphQL.

**Comment by akofink** [comment]
Thanks, @bastilian! Closing this one in favor of https://github.com/RedHatInsights/compliance-frontend/pull/845.

**Pull request closed without merging**